docs/blog/2018-07-16-swag-store/index.md

```diff
@@ -0,0 +1,80 @@
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">What a Saturday afternoon 😁<br><br>Received them <a href="https://twitter.com/hashtag/Gatsbyjs?src=hash&amp;ref_src=twsrc%5Etfw">#Gatsbyjs</a> <a href="https://twitter.com/hashtag/swag?src=hash&amp;ref_src=twsrc%5Etfw">#swag</a>😍🤩<br><br>Thank you <a href="https://twitter.com/jlengstorf?ref_src=twsrc%5Etfw">@jlengstorf</a> and the team behind it for making it happen. <a href="https://twitter.com/hashtag/buildwithgatsby?src=hash&amp;ref_src=twsrc%5Etfw">#buildwithgatsby</a> <a href="https://t.co/B7jHQi6e8Y">pic.twitter.com/B7jHQi6e8Y</a></p>&mdash; Aman Mittal 🖖⚛️☕ (@amanhimself) <a href="https://twitter.com/amanhimself/status/1023124667446509570?ref_src=twsrc%5Etfw">July 28, 2018</a></blockquote>
+
+## How the store works
+
+All of the source code for the swag store [has been open sourced][swag-source]. The front-end is a [Gatsby v2][v2] site (naturally) using [Shopify][shopify] (and their [JS Buy SDK][js-buy-sdk]) to display products and manage carts, [Auth0][auth0] to handle authentication, and a [custom Express API][swag-api] to manage discount code requests.
+
+We also use [Peril][peril] to [automatically invite contributors][invite] to the Gatsby organization on GitHub and post details about claiming swag. Here’s an example (and some kind words from Gatsby contributor [Payton Burdette](https://twitter.com/beardedpayton)) showing what the automated response looks like:
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr"><a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> is killing it in open source. This is how you treat contributors. Take notes 📝😊 <a href="https://t.co/9ta3IvfiZF">pic.twitter.com/9ta3IvfiZF</a></p>&mdash; Payton Burdette (@beardedpayton) <a href="https://twitter.com/beardedpayton/status/1020339698332512256?ref_src=twsrc%5Etfw">July 20, 2018</a></blockquote>
+
+## What’s next?
+
+We have a lot of ideas for the Gatsby store — from [new swag ideas](https://github.com/gatsbyjs/store.gatsbyjs.org/issues?q=is%3Aissue+is%3Aopen+label%3A%22Swag+Ideas%22+sort%3Aupdated-desc) to adding more rewards for contributors who go above and beyond — but we’d love to hear more from you! Please [tweet at us][twitter] or open issues on [the store repo][swag-source] with ideas.
+
+For now, that’s it! We’ll leave you with my our favorite genre of tweet: “dogs wearing Gatsby swag”.
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Thanks <a href="https://twitter.com/gatsbyjs?ref_src=twsrc%5Etfw">@gatsbyjs</a> for the t-shirt! If there are any developers out there that haven’t tried out Gatsby yet here is my dog Birch silently judging you. 😉 Trust me, it’s awesome. <a href="https://t.co/THDSmhKdBb">pic.twitter.com/THDSmhKdBb</a></p>&mdash; Ryan Wiemer (@ryanwiemer) <a href="https://twitter.com/ryanwiemer/status/1022267081998266368?ref_src=twsrc%5Etfw">July 25, 2018</a></blockquote>
```

Comment:
wondering if this and other quotes tweets will show the image in final post? It'd be cool to see this dog w/o having to click on a link!

Reply:
Once the preview finishes building, it _should_ show the images. I checked the build locally and it was working, so we'll see if that stays true. 😅